staging/src/k8s.io/kubectl/pkg/cmd/config/config.go

```diff
@@ -18,6 +18,7 @@ package config
 
 import (
 	"fmt"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
Go native packages are in a group and k8s related packages should be in different group;

```suggestion
        "fmt"
        "path"
        
	"k8s.io/cli-runtime/pkg/genericclioptions"
```